libs/application/templates/transport-authority/transfer-of-vehicle-ownership/src/fields/ValidationErrorMessages/index.tsx

```
@@ -94,15 +100,13 @@ export const ValidationErrorMessages: FC<
                     error?.errorNo
 
                   return (
-                    <li>
-                      <Text variant="small">
-                        {message || defaultMessage || fallbackMessage}
-                      </Text>
-                    </li>
+                    <Bullet>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)